SymPEP-0001.md

```diff
@@ -0,0 +1,222 @@
+SymPEPs but apply to areas other than the SymPy library itself. They may
+propose an implementation, but not to SymPy’s codebase; they require community
+consensus. Examples include procedures, guidelines, changes to the
+decision-making process, and changes to the tools or environment used in SymPy
+development. Any meta-SymPEP is also considered a Process SymPEP.
+
+## Purpose
+
+The SymPEP process and corresponding documents serve several purposes:
+
+- To decide, as a community, whether a proposed change should be made.
+- To decide on the process by which a change will be implemented, for example,
+  whether the change should be implemented in several stages, whether
+  intermediate releases are required for things like deprecations, and so on.
+- To document the above discussions and decisions for future reference.
+- To document the motivations for a change from the perspective of end users
+  who may be affected by it. Here "end users" means both users of the SymPy
+  library, as well as developers working on parts of SymPy itself which may be
+  affected by the change.
+
+Importantly, **a SymPEP is not documentation** for the proposed change. End
+user documentation should be included with the implementation of the feature
+in the corresponding SymPy documentation. This also means that other
+documentation should not cross-reference a SymPEP as if it were documentation
+for a feature. Even technical discussion of a feature should be documented
+separately from a SymPEP. The reason is that SymPEPs will necessarily include
+details that are irrelevant to the final implementation, such as discussions
+of alternate implementations which were rejected and discussions of
+implementation of the change.
+
+## SymPEP Workflow
+
+Discussions around a proposed change may begin informally on the mailing list
+or SymPy issue tracker. However, once it is decided that the formal SymPEP
+process is desired for a change, the discussion should move to the
+[SymPEP repository](https://github.com/sympy/SymPEPs) on GitHub.
+
+Every SymPEP must have a champion. This is a person who is responsible for
+writing the SymPEP, leads the discussion around it, and tries to create
+consensus around it.
+
+The author of a SymPEP should fork the repository and create a pull request
+with a new SymPEP document based on the [SymPEP template](SymPEP-template).
+The SymPEP document may be named `SymPEP-XXXX.md` until a number is assigned.
+One of the core SymPy developers will then assign a number to the SymPEP, in
+which case `XXXX` should be replaced with the number with leading 0s. The
+person who assigns the number should also update the
+[README](https://github.com/sympy/SymPEPs/blob/main/README.md) of the main
+[SymPEP repository](https://github.com/sympy/SymPEPs) to list that number.
+Numbers should be assigned to SymPEPs as soon as it is determined that they
+are a legitimate proposal. If a SymPEP ends up being rejected or postponed, it
```

Comment:
This sounds like the assignment of a number indicates whether the PEP is a draft or a proposal. Who decides this and when does the number get assigned?